guide/03-the-gis/cloning-content.ipynb

```diff
@@ -0,0 +1,2051 @@
+{
```

Comment:
<p>The step to validate if a web map is successfully cloned from source to target is great here. Is there a way to also find out that the web map can be opened correctly, and the links of the containing FS are not broken?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
<p>I added code cells visualizing one of the source webmaps and the corresponding target webmaps. Does that verification seem solid?</p>